Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/Image.hs

```diff
@@ -136,19 +143,22 @@ validateImage isDashboard (personId, _, merchantOpCityId) ImageValidateRequest {
         Nothing -> throwError $ RCMandatory (show imageType)
       else return Nothing
 
+  images <- filter (uncurry (||) . (isNothing &&& (/= workflowTransactionId)) . (.workflowTransactionId)) <$> Query.findRecentByPersonIdAndImageType personId imageType
   unless isDashboard $ do
-    images <- Query.findRecentByPersonIdAndImageType personId imageType
     transporterConfig <- findByMerchantOpCityId merchantOpCityId (Just (DriverId (cast personId))) >>= fromMaybeM (TransporterConfigNotFound merchantOpCityId.getId)
     let onboardingTryLimit = transporterConfig.onboardingTryLimit
-    when (length images > onboardingTryLimit) $ do
+    when (length images > onboardingTryLimit * bool 1 2 (imageType == DVC.AadhaarCard)) $ do
```

Comment:
Move `onboardingTryLimit` to documentVerificationConfig.

Reply:
This config is getting accessed in lot of places (in previous code) where vehicleCategory we don't have yet.